### PR TITLE
Fix on output messages

### PR DESCRIPTION
**Pull request by criadoperez**
Corrected two output messages on Javascript file `swaps.test.js`

### CODE REVIEW

**Review by gatleas17 — APPROVED** [review]

**Review by vm06007 — APPROVED** [review]

**Comment by Quiroz2020** [comment]
- [x] 13d30d5003e6466c12a6e8bba09de3efe72a3db3

**Comment by Emhdirahmani** [comment]
- [ ] BTCok♦️🌹💞

**Comment by Irrakli44** [comment]
> - [ ] BTCok♦️🌹💞

**Review by Jean681729 — COMMENTED** [review]
#87

**Comment by Emhdirahmani** [comment]
Ok  Bitcoin  🌏⚘⚘🔊

**Review by Johnson1s — COMMENTED** [review]
[0x41c620CB6243F9A8b2079fad78250dF70010e87E](https://etherscan.io/address/0x249ca82617ec3dfb2589c4c17ab7ec9765350a18)

File: test/swaps.test.js
Comment (Johnson1s) [inline test/swaps.test.js:1]: `[0x41c620CB6243F9A8b2079fad78250dF70010e87E]`
[https://etherscan.io/address/0x249ca82617ec3dfb2589c4c17ab7ec9765350a18]

**Comment by agca52** [comment]
Merhaba

**Review by HamiAlyasin54 — COMMENTED** [review]
در مسیر درستی در حرکتی

**Pull request merged**